modules/ossm-security-cert-manage.adoc

```diff
@@ -20,7 +21,7 @@ To use an existing signing (CA) certificate and key, you must create a chain of
 
 Add the certificates to {ProductShortName} by following these steps. Save the example certificates from the link:https://github.com/maistra/istio/tree/maistra-{MaistraVersion}/samples/certs[Maistra repository] locally and replace `<path>` with the path to your certificates.
 
-. Create a secret `cacert` that includes the input files `ca-cert.pem`, `ca-key.pem`, `root-cert.pem` and `cert-chain.pem`.
+. Create a secret named `cacert` that includes the input files `ca-cert.pem`, `ca-key.pem`, `root-cert.pem` and `cert-chain.pem`.
```

Comment:
Line 22 - This is the second time I've walked through this and missed the line about downloading the sample files.  Can we make step 1 "Save the example certficates....."

Reply:
edit made